### PR TITLE
add readme doc

### PR DESCRIPTION
**Pull request by carmecass**

### CODE REVIEW

File: READEME.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# HELLO WORLD
+
+## Intro
```
Comment (alicia-alcaide) [inline READEME.md]: @carmecass "Intro" no, but "Introduction" yes

File: READEME.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+TODO explain project
```
Comment (alicia-alcaide) [inline READEME.md]: @carmecass, you should finish this

File: READEME.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+## Functional Description
+
+TODO describe functionality
```
Comment (alicia-alcaide) [inline READEME.md]: @carmecass, you should finish this

File: READEME.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+## Technical Description
+TODO  desceibe specs
```
Comment (alicia-alcaide) [inline READEME.md]: @carmecass you should finish this

File: READEME.md (hunk 1 above)
Comment (carmecass) [reply READEME.md]: @alicia-alcaide  sorry but "intro" is ok

**Pull request merged**